inc/Classes/Func.php

```diff
@@ -436,85 +436,92 @@ public function escape_sql($text)
     public function text2html($string, $mode = 0)
     {
         global $db;
+        
```

Comment:
Can we have a quick documentation / inline comment about what `$mode` is and what possible values it can have?

Alternative: Replacing the magic numbers (0, 4, etc.) with constants that have a descriptive name.

Reply:
I'll try to remember what this was about and add a bit of documentation in that regard